virtcontainers/clh.go

```diff
@@ -455,9 +455,13 @@ func (clh *cloudHypervisor) resizeVCPUs(reqVCPUs uint32) (currentVCPUs uint32, n
 			"reqVCPUs":              reqVCPUs,
 			"configDefaultMaxVCPUs": clh.config.DefaultMaxVCPUs,
 			"clhMaxVCPUs":           info.Config.Cpus.MaxVcpus,
-		}).Debug("exceeding the maxVCPUs")
-		return currentVCPUs, newVCPUs, fmt.Errorf("Cannot resize vCPU to %d: exceeding the maximum amount of vCPUs (%d or %d)",
-			reqVCPUs, clh.config.DefaultMaxVCPUs, info.Config.Cpus.MaxVcpus)
+		}).Warnf("exceeding the maxVCPUs (resizing to maxVCPUs)")
```

Comment:
- Nit: This doesn't need to be `Warnf()` as there are no formats in the string, so you could make it a `Warn()`.
- `maxVCPUs` isn't a field that is being logged, so the user won't know what value is being used. I think this message probably wants to mention `clhMaxVCPUs` instead?